src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Company.WebApplication1
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
Are the cools kids doing `async Main` these days? Didn't we change other templates to do this?

Reply:
Not yet, there's an issue with debugging, but we need to bite the bullet and just do it across the board.

Reply:
sounds like i should not change it to async given @davidfowl's comments that there is an issue with debugging, so i'll leave as is unless asked to change it. easy either way.